snakemake/executors/__init__.py

```diff
@@ -1913,7 +1915,7 @@ def run(self, job, callback=None, submit_callback=None, error_callback=None):
         # request resources
         container.resources = kubernetes.client.V1ResourceRequirements()
         container.resources.requests = {}
-        container.resources.requests["cpu"] = job.resources["_cores"]
+        container.resources.requests["cpu"] = "{}m".format(int(job.resources["_cores"] * self.k8s_cpu_scalar * 1000))
```

Comment:
Can job.resources["_cores"] ever be non-numeric? e.g. a function? Or have any functions been evaluated before it gets to this point?

Reply:
No, it is always an integer.

Reply:
great! thanks